### PR TITLE
Fixes #21952: Documentation for FQDN overriding

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/21952

### CODE REVIEW

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+FQDN is an internal name, or it's incorrectly set-up by OS or not found in a normalized format by the agent. 
+
+This can be problematic and for example break the possibility for Rudder to trigger an agent run from node's 
+compliance page.
+
+In these cases, you can override the FQDN from the node thanks to a xref:extend-nodes-inventory[node inventory extension].
+
+The inventory extension must define the reserved property key `rudder_override_hostname` with a string value.
+
+If that key is defined, it is prefered to other values retrieved in agent inventory (`<RUDDER><HOSTNAME>` or else
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:16]: preferred

File: src/reference/modules/usage/pages/advanced_node_management.adoc [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+
+[NOTE]
+
+====
+The value used for overriding FQDN must be legit https://datatracker.ietf.org/doc/html/rfc1123#page-13](https://datatracker.ietf.org/doc/html/rfc1123#page-13[as of RFC1123 (section 2.1)]
+else it won't be used in place of the original one.
+====
+
+Example inventory hook script to put on the node so that the node will be
+see on Rudder with the FQDN `"node1-overridden.rudder.local.override`:
```
Comment (ElaadF) [inline src/reference/modules/usage/pages/advanced_node_management.adoc:30]: remove the extra "

**Review by ElaadF — CHANGES_REQUESTED** [review]

**Comment by ncharles** [comment]
it misses the version in which the feature appeared in Rudder

**Comment by ElaadF** [comment]
> it misses the version in which the feature appeared in Rudder

Can you open a ticket ?
This PR is obsolete, I will close it, the real one is here: https://github.com/Normation/rudder-doc/pull/931

**Pull request closed without merging**